lib/graphiti/schema.rb

```diff
@@ -166,6 +166,8 @@ def flag(value)
     def sorts(resource)
       {}.tap do |s|
         resource.sorts.each_pair do |name, sort|
+          next unless resource.attributes[name][:schema]
```

Comment:
This seems fine to me, unless you can envision a scenario where we allow something like the sort in the schema but not filtering/reading/etc.

Reply:
My only use case is that the whole attribute with it's sorting and filtering be hidden. My current project has a table with dynamic columns that changes with customer and year to year. So they are not relevant or available in different contexts. The place where this is hurting is in tests where I create test data that gets added to the schema, which never makes sense in production.